src/test/java/software/amazon/cloudformation/resource/ValidatorTest.java

```diff
@@ -323,6 +323,61 @@ public void validateDefinition_invalidHandlerSection_shouldThrow() {
             .withNoCause().withMessage("#/handlers/read: required key [permissions] not found");
     }
 
+    @ParameterizedTest
+    @ValueSource(ints = { 1, 721 })
+    public void validateDefinition_invalidTimeout_shouldThrow(final int timeout) {
+        // modifying the valid-with-handlers.json to add invalid timeout
+        final JSONObject definition = new JSONObject(new JSONTokener(this.getClass()
+            .getResourceAsStream("/valid-with-handlers.json")));
+
+        final JSONObject createDefinition = definition.getJSONObject("handlers").getJSONObject("create");
+        createDefinition.put("timeoutInMinutes", timeout);
+
+        final String keyword = timeout == 1 ? "minimum" : "maximum";
+
+        assertThatExceptionOfType(ValidationException.class).isThrownBy(() -> validator.validateResourceDefinition(definition))
+            .withNoCause().withMessage(
+                String.format("#/handlers/create/timeoutInMinutes: failed validation constraint for keyword [%s]", keyword));
```

Comment:
`.withMessageContaining("#/handlers/create/timeoutInMinutes")` is fine. the reason we check the message at all is to make it easy to debug in future if the test fails for any other schema error. if you really want, `.withMessageContaining("#/handlers/create/timeoutInMinutes").withMessageContaining(keyword)` is extra good. but `.withMessage` is too brittle, requires too much maintenance.

Reply:
will do